heiko/migrate.py

```diff
@@ -1,18 +1,38 @@
 import sqlite3
 import getpass
 from heiko.utils import log
+import os
 
 
-def migrate_user(auth, client):
+def migrate_user(auth, client, cfgobj):
     """
     Migration function that reads data from old matomat sqlite and creates user in the backend
 
     :auth: dict
+    :client: userauth object
+    :cfgobj: config dict
     :returns: bool
     """
 
-    sq = sqlite3.connect('/home/heiko/matomat.db')
+    # Check if configured database is there and can be read
+    try:
+        migration_database = cfgobj["accouting"]["migration_database"]
+        if os.access(migration_database, os.R_OK):
+            matomat_db = sqlite3.connect(migration_database)
+        else:
+            raise IOError
+    except (IOError, KeyError):
+        log("Migration database cannot be found or is not configured. Aborting.", serv="ERROR")
+        return False
+
+    # First, ask for Admin yes/no
```

Comment:
I would rather ask this question, after the user was actually found and we confirmed that we want to migrate her.

Reply:
Mh, yes initially i was thinking the same, but most of the users are typing their usernames by themselves. 

The goal was that the user does not touch the keyboard until the admin question was answered by the administartor. 

What do you think?